libfreerdp/core/nego.c

```diff
@@ -1219,6 +1219,11 @@ void nego_set_gateway_bypass_local(rdpNego* nego, BOOL GatewayBypassLocal)
 	nego->GatewayBypassLocal = GatewayBypassLocal;
 }
 
+void nego_set_gateway_access_token(rdpNego* nego, char* token)
+{
+	nego->GatewayAccessToken = _strdup(token);
```

Comment:
add a `NULL` check, if possible.

Reply:
Is this function used anywhere?

Reply:
Very sorry for this. Did not clean up as well as i thought. Will remove this function.

Reply:
@jaccobraat Don't worry, stuff happens, that is what a review is for ^^